filebeat/tests/system/test_registrar.py

```diff
@@ -161,6 +162,36 @@ def test_custom_registry_file_location(self):
 
         assert os.path.isfile(os.path.join(self.working_dir, "a/b/c/registry"))
 
+    def test_registry_file_permissions(self):
+       """
+       Test that filebeat registry permission is set as per configuration
+       """
+
+       self.render_config_template(
+           path=os.path.abspath(self.working_dir) + "/log/*",
+           registryFile="a/b/c/registry",
+           registryFilePermission = 0600
```

Comment:
Could you add one more tests which sets the registry file to the non default permission?